packages/web-components/src/theme/index.ts

```diff
@@ -1,2 +1,387 @@
-export * from './design-tokens.js';
+export {
```

Comment:
we should eventually add a test which verifies that this file exports everything what is exported from `design-tokens`